content/en/dynamic_instrumentation/_index.md

```diff
@@ -23,118 +23,162 @@ further_reading:
+**Note**: The "Dynamic Instrumentation Read Configuration" (`debugger_read`) permission is required to access the Dynamic Instrumentation page, and the "Dynamic Instrumentation Write Configuration" (`debugger_write`) permission is required to create or modify instrumentations. For more information about roles and on how to assign roles to users, see [Role Based Access Control][8].
 
 ### Create a logs index
 
-Dynamic Instrumentation logs are sent to Datadog and they appear alongside your application logs.
+Dynamic Instrumentation allows you to create "Dynamic Logs" that are sent to Datadog and appear alongside your regular application logs.
 
 If you use [Exclusion filters][9], ensure Dynamic Instrumentation logs are not filtered:
 
 1. Create a logs index and [configure it][10] to the desired retention with **no sampling**.
 2. Set the filter to match on `source:dd_debugger` (all Dynamic Instrumentation logs have this source).
 3. Ensure that the new index takes precedence over any other with filters that match that tag, because the first match wins.
 
-### Enable Dynamic Instrumentation
+## Enable Dynamic Instrumentation
+
+To enable Dynamic Instrumentation on a service, go to the [in-app setup page][16]. 
 
-To enable Dynamic Instrumentation on a service, select its runtime and follow the setup instructions:
+For more details instructions, select the relevant runtime below:
 
 {{< partial name="dynamic_instrumentation/dynamic-instrumentation-languages.html" >}}
 
 ## Explore Dynamic Instrumentation
 
 Dynamic Instrumentation can help you understand what your application is doing at runtime. By adding a Dynamic Instrumentation probe you are exporting additional data from your application, without the need to do any code change or redeployment.
 
-### Creating a probe
+# Using Probes
+
+A probe allows you to collect data from specific points in your code without halting the execution of the program. 
+
+If you're a SRE, you can think of probes as a way of enhancing your observability by adding dynamic logs, metrics, and spans to running application without needing to stop or restart anything. This meant you can gather data immediately without disturbing the user experience or requiring lengthy deployments.
 
-Both log and metric probes require the same initial setup:
+If you're a developer, you can think of a probe as a "non-breaking breakpoint". In traditional debugging, a breakpoint is a point in the program where the execution stops, allowing the developer to inspect the state of the program at that point. However, in real-world production environments, it's not practical or even possible to stop the execution of the program. Probes fill in this gap by allowing you to inspect variable state in production environments in a completely non-intrusive way.
```

Comment:
Hmm, I'm not sure we should call out the roles here directly. 🤔 I think this is how we think about the personae internally, but do we want to bleed that into the docs? If you're a DevOps, what is DI to you?

Reply:
That's fair, I removed SRE but kept developer in ad899feacc7cef6e2051a3430fc4d5da7daf872b to make this more fluid, does it make sense now?